modules/util/portal-tools-rest-builder-test-impl/rest-openapi.yaml

```diff
@@ -1981,7 +1981,7 @@ paths:
             tags: ["ERCSiteTestEntity"]
     "/sites/{siteExternalReferenceCode}/erc-site-test-entities/{ercSiteTestEntityExternalReferenceCode}/permissions":
         get:
-            operationId: getSiteERCSiteTestEntityPermisssionsPage
```

Comment:
I think this was incorrect when creating the endpolint in the previous commit. Please squash this single change within the previous commit